release-team/role-handbooks/test-infra/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+  - test-infra-role
```

Comment:
(assume this is accurate?)

Reply:
It will map to https://github.com/kubernetes/sig-release/blob/master/OWNERS_ALIASES#L38 
so seems fine.